tracer/build/_build/Build.Steps.cs

```diff
@@ -299,7 +299,7 @@ partial class Build
             }
             else
             {
-                var (architecture, ext) = GetUnixArchitectureAndExtention();
+                var (architecture, ext) = GetUnixArchitectureAndExtension(includeMuslSuffix: true);
```

Comment:
Should be IsAlpine, not true, I think.

Reply:
It doesn't, but I tweaked the switch in https://github.com/DataDog/dd-trace-dotnet/pull/1808/commits/16a3842fbe57473c09fe78732aa221397ba3dcf6